src/locales/en/en.json

```diff
@@ -86,10 +86,13 @@
       }
     },
     "upload": {
-      "text": "Search with a bed file or csv file"
+      "text": "Search with a csv file"
     }
   },
   "panelResult": {
+    "title": {
+      "text": "Panel Results"
```

Comment:
Duplicate. The value already exists on line 98. Use that line instead.

Reply:
done